app/services/phone_number_capabilities.rb

```diff
@@ -9,6 +9,14 @@ def self.load_config
 
   attr_reader :phone, :phone_confirmed
 
+  def self.translated_international_codes
+    translated_international_codes_data = {}
+    INTERNATIONAL_CODES.each do |k, value|
+      value['name'] = I18n.t("countries.#{k}")
+      translated_international_codes_data[k] = value
```

Comment:
should this be a `.merge` so that it creates a copy instead of mutates the constant?
```suggestion
      translated_international_codes_data[k] = value.merge('name' => I18n.t("countries.#{k}"))
```